src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/adminappealwithdrawn/AdminAppealWithdrawnAboutToSubmitHandler.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.hmcts.reform.sscs.ccd.presubmit.adminappealwithdrawn;
+
+import static java.util.Objects.requireNonNull;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.sscs.ccd.callback.Callback;
+import uk.gov.hmcts.reform.sscs.ccd.callback.CallbackType;
+import uk.gov.hmcts.reform.sscs.ccd.callback.PreSubmitCallbackResponse;
+import uk.gov.hmcts.reform.sscs.ccd.domain.EventType;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseData;
+import uk.gov.hmcts.reform.sscs.ccd.presubmit.PreSubmitCallbackHandler;
+
+
+@Service
+@Slf4j
+public class AdminAppealWithdrawnAboutToSubmitHandler implements PreSubmitCallbackHandler<SscsCaseData> {
+
+    @Override
+    public boolean canHandle(CallbackType callbackType, Callback<SscsCaseData> callback) {
+        requireNonNull(callback, "callback must not be null");
+        requireNonNull(callbackType, "callbacktype must not be null");
+
+        return callbackType.equals(CallbackType.ABOUT_TO_SUBMIT)
+                && callback.getEvent() == EventType.ADMIN_APPEAL_WITHDRAWN;
+    }
+
+    @Override
+    public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType, Callback<SscsCaseData> callback, String userAuthorisation) {
+        if (!canHandle(callbackType, callback)) {
+            throw new IllegalStateException("Cannot handle callback");
+        }
+
+        SscsCaseData caseData = callback.getCaseDetails().getCaseData();
+
+        log.info("Setting InterlocReviewState to N/A for case id {} "
```

Comment:
The log message doesn't match what it is actually doing? Can we update so this says 'Setting InterlocReviewState to null for case id {} ' - This is in every handler

Reply:
Updated in every handler